l1infotreesync/processor.go

```diff
@@ -276,54 +276,9 @@ func (p *processor) GetProcessedBlockUntil(ctx context.Context, blockNum uint64)
 	return processedBlockNum, hash, nil
 }
 
-// Reorg triggers a purge and reset process on the processor to leaf it on a state
-// as if the last block processed was firstReorgedBlock-1
+// Reorg is intentionally left as a no-op. This method is retained for compatibility
+// with the processor interface and to allow for potential future implementation.
 func (p *processor) Reorg(ctx context.Context, firstReorgedBlock uint64) error {
```

Comment:
Maybe we can leave the function as it was, if in the future we decide we will need the reorg behavior on L1 syncers again. This way, the code changed on the PR will be really small, plus, we leave the support for the reorgs in the syncer. But, I do not have a strong opinion on this, so feel free to do what makes more sense to you.

Reply:
leaving it as nil as it will be clear what this is doing
we can pull code from this PR if necessary